### ISSUE TITLE
Depreciation should be able to be set as the asset level rather than the model level

### ISSUE THREAD

**Issue by Jarli01** [snipe/snipe-it#6697]
**Server (please complete the following information):**
 - Snipe-IT Version 4.6.8
 - OS: [e.g. Ubuntu, CentOS] CentOS
 - Web Server: [e.g. Apache, IIS] Apache
 - PHP Version 7.1.26

**Is your feature request related to a problem? Please describe.**
When applying depreciation to an asset model the function can get a bit murky, as you may not generally depreciate an individual item that is under say $1000. But you would depreciate an order of that type of item that exceeds $1000 total. 

IE: 1 Monitor cost $200 - don't depreciate
20 Monitors cost $200x20=$4000 do depreciate this order of monitors. 

What the current system would have you do to avoid this is to create multiple asset models of the similar name. One that has depreciation and one that doesn't. 

This mucks up the entire database with unnecessary models and just makes things confusing in general.  

**Describe the solution you'd like**
I'd like to ask that depreciation can be assigned per asset individually rather than at the asset model level. 

While generally having depreciation set at the asset model should work, the practicality of it is that many accounting departments won't depreciate a model but a set of assets based on a dollar value. 

Since there isn't a means of adding a depreciation to an order system of snipe-it adding this functionality to the asset itself is the best approach I can think of as it allows for the granular depreciation that actually is taking place. 

**Describe alternatives you've considered**
Creating multiple assets, one for an asset model that was purchased exceeding a certain dollar value total and having depreciation assigned to it. 

And another asset model that is for "individual one off purchases" that don't exceed a dollar value. Meaning duplicate asset models.

**Comment by Jarli01** [comment]
This is a critical system function that our accounting team will require, as adding multiple asset models with different names isn't a practical solution to this problem. 

Since the asset model isn't changing, just how the asset has been purchased and the total value. 

Having multiple models doesn't work, and makes searching through the system more cumbersome since 1 asset may be depreciating and another may not even if they are purchased only a week apart.

**Comment by Jarli01** [comment]
Since there is an "Order Number" field that is searchable, having depreciation applied to the asset makes even more sense than at the model level. 

Since you can't search for an model by order number. The Model should be a _unique_ field for all critical functions and not require having different models for the same asset type.

**Comment by Nicknack1234** [comment]
I agree, The depreciation only makes sense on the assets themselves to me rather than the models. We have got around this by just adding a fieldset with a depreciation field.

**Comment by Jarli01** [comment]
> We have got around this by just adding a fieldset with a depreciation field.

Is math being performed against this fieldset or are you having to do additional operations for that?

**Comment by Jarli01** [comment]
Please keep this request open.

**Comment by Jarli01** [comment]
Yes it is

**Comment by zebby-nick** [comment]
Yes

**Comment by Jarli01** [comment]
Yes

**Comment by jonarcgit** [comment]
is this being considered at all?

**Comment by Jarli01** [comment]
Yes

**Comment by Jarli01** [comment]
Yes

**Comment by Jarli01** [comment]
Yes

**Comment by Jarli01** [comment]
Yes

**Comment by Jarli01** [comment]
Yes

**Comment by Jarli01** [comment]
> I agree, The depreciation only makes sense on the assets themselves to me rather than the models. We have got around this by just adding a fieldset with a depreciation field.

Nick what is the custom fieldset you added, can you show me?
